app/translations/en.json

```
@@ -73,6 +73,8 @@
                 "uploadingErrorMessageSuffix": "blueprint upload error"
             }
         },
+        "welcomeText": "Welcome to Cloudify: your open source,\nmulti-cloud orchestration platform.\n\nWe’ll start by setting up your cloud environment in\nthe following easy steps. Ready? Let’s go.\n\nNote: this process can take up to 5 minutes, and you\nwill need your API access keys.",
```

Optional:

I'm not an expert in that domain, but I suspect that a bit better and easier to read would be to have (line endings a bit different):
```suggestion
        "welcomeText": "Welcome to Cloudify: your open source,\nmulti-cloud orchestration platform.\n\nWe’ll start by setting up your cloud environment\nin the following easy steps. Ready? Let’s go.\n\nNote: this process can take up to 5 minutes,\nand youwill need your API access keys.",
```

Welcome to Cloudify: your open source, 
multi-cloud orchestration platform.

We’ll start by setting up your cloud environment
in the following easy steps. Ready? Let’s go.

Note: this process can take up to 5 minutes, 
and you will need your API access keys.

---

Definitely a good idea - changed.